src/pages/About/About.js

```diff
@@ -0,0 +1,149 @@
+// @flow strict
+
+import React, { Component } from 'react'
+import { withI18n } from '@lingui/react'
+import { type I18n } from '@lingui/core'
+
+import { TitleHeader } from 'components'
+import { gaSendException } from 'utils/analytics'
+
+import {
+  JIcon,
+  JLink,
+  JLoader,
+  AppLogo,
+} from 'components/base'
+
+import styles from './about.m.scss'
+
+type Props = {|
+  +i18n: I18n,
+|}
+
+type StateProps = {|
+  +version: ?string,
+|}
+
+const VERSION_TXT_URL: string = 'version.txt'
+
+class AboutView extends Component<Props, StateProps> {
+  constructor(props: Props) {
+    super(props)
+
+    this.state = {
+      version: null,
+    }
+  }
+
+  async componentDidMount() {
+    try {
+      this.setState({ version: await this.requestVersion() })
+    } catch (error) {
+      const errorMessage: string = error.message
+
+      console.error(errorMessage)
+
+      gaSendException({
+        exDescription: errorMessage,
+        exFatal: false,
+      })
+
+      const { i18n }: Props = this.props
+
+      this.setState({
+        version: i18n._(
+          'About.version.error',
+          null,
+          { defaults: 'App version request failed. Please, try again a bit later' },
+        ),
```

Comment:
Let's shorten it to just `unavailable`.

It will be enough for user to understand something went wrong, yet it takes a lot less space (and is a lot more bulletproof for translation).

Reply:
(that's about text itself)